engine/menus/pokedex.asm

```diff
@@ -489,7 +489,7 @@ ShowPokedexDataInternal:
 	pop af
 	ld [wd11e], a
 	ld a, [wcf91]
-	ld [wd0b5], a
+	ld [wd0b5], a ; equivalent to wd0b5 = wcf91
```

Comment:
These `wXXXX` labels should/will eventually have meaningful names, making comments unnecessary.